index.js

```diff
@@ -0,0 +1,88 @@
+const compose = require('koa-compose');
+const os = require('os');
+const path = require('path');
+const appRootPath = require('app-root-path');
+const package_json = require(appRootPath + path.sep +'package.json');
+
+const HEALTH_PATH = '/health';
+const ENV_PATH = '/env';
+const INFO_PATH = '/info';
+const METRICS_PATH = '/metrics';
+const SECURE_PROP_NAMES = ['admin', 'user', 'password', 'pass', 'pwd', 'login', 'username'];
+
+/**
+ * Writes {status: 'UP'} to response body if request path is /health
+ */
+async function health(ctx, next) {
+  if (HEALTH_PATH == ctx.path)
+    ctx.body = {status: 'UP'};
+  else
+    await next();
+}
+
+/**
+ * Exposes application and resources information. E.g. name, version.
+ */
+async function info(ctx, next) {
+  if (INFO_PATH == ctx.path)
+    ctx.body = {build:
+      {
+        version: package_json.version,
+        name: package_json.name,
+        main: package_json.main,
+        description: package_json.description
+      }
+    };
+  else
+    await next();
+}
+
+/**
+ * Exposes environment properties. Secure variables (such as 'user', 'password', 'pass' etc are) values will be hidden.
+ */
+async function env(ctx, next) {
+  if (ENV_PATH == ctx.path) {
+    const envCopy = JSON.parse(JSON.stringify(process.env)); //deep copy
```

Comment:
Since `process.env` is a flat structure, you could do shallow copy: `Object.assign({}, process.env)`.